FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -45,6 +47,37 @@ def initializeHandler( cls, serviceInfo ):
       client = SystemAdministratorClient( 'localhost' )
       gThreadScheduler.addPeriodicTask( 60, client.storeHostInfo )
 
+    # Check the flag for dynamic monitoring
+    dynamicMonitoring = cls.srv_getCSOption( 'DynamicMonitoring', False )
+
+    if dynamicMonitoring:
+      client = SystemAdministratorClient( 'localhost' )
```

Comment:
What I understand is that the service collect information about all component  only the localhost. In principle localhost works in linux.